stream-chat-android-ui-components-sample/src/main/res/layout/fragment_user_login.xml

```diff
@@ -85,4 +85,13 @@
         app:layout_constraintStart_toStartOf="parent"
         tools:text="@string/sdk_version_template"
         />
+
+    <io.getstream.chat.android.ui.images.AttachmentGallery
```

Comment:
Why are we adding it to user login fragment?

Reply:
Doesn't make any sense. I'll remove this in my next commit.